README.md

```diff
@@ -1,70 +1,62 @@
-# Getting Started with Create React App
+# Stock Company Information
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+> This is a single page web application that retrieves real live data from the Financial Modeling Prep API. Users can get information on their favourite company at Dow Jones Industrial Avarage.
+> Companies are displayed in the Home Page. Information which inclujde the stock index is displayed at the company information section.
 
-## Available Scripts
+![screenshot](./home.png)
+![screenshot](./company.png)
 
-In the project directory, you can run:
+## Live Demo
 
-### `npm start`
+You can see [the app live here](https://aduda-boaz.github.io/finance-modelling/)
```

Comment:
- [ ] Kindly make sure your live version works properly. You can also use netlify to deploy your website.